cmd/network-tracer/tracer.go

```diff
@@ -74,13 +75,15 @@ func (nt *NetworkTracer) Run() {
 	httpMux.HandleFunc("/status", func(w http.ResponseWriter, req *http.Request) {})
 
 	httpMux.HandleFunc("/connections", func(w http.ResponseWriter, req *http.Request) {
+		now := time.Now()
 		cs, err := nt.tracer.GetActiveConnections(getClientID(req))
 		if err != nil {
 			log.Errorf("unable to retrieve connections: %s", err)
 			w.WriteHeader(500)
 			return
 		}
 		writeConnections(w, cs)
+		log.Infof("Got request on /connections: retrieved %d connections in %+v", len(cs.Conns), time.Now().Sub(now))
```

Comment:
This can get pretty noisy. Can we do what the process-agent does and after 5m only log this every 2 minutes (or N iterations)?